src/components/utils/errors.js

```diff
@@ -0,0 +1,8 @@
+export const errors = {
```

Comment:
Yes, good point, it might come in handy when will work on the internationalisation.